app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -0,0 +1,270 @@
+}
+
+/**
+ * Start liveness capture
+ */
+type AcuantStartSelfieCapture = (callback: (nextImageData: string) => void) => void;
+interface AcuantPassiveLivenessInterface {
+  startSelfieCapture: AcuantStartSelfieCapture;
+}
+
+declare global {
+  interface Window {
+    /* Document load callback to assign Javascript Web SDK globals */
+    loadAcuantSdk: () => void;
+    /* Acuant configuration */
+    acuantConfig: AcuantConfig;
+    /* Acuant Passive Liveness API */
+    AcuantPassiveLiveness: AcuantPassiveLivenessInterface;
+    /* Possible AcuantJavascriptWebSdk on the window object (11.5.0) */
+    AcuantJavascriptWebSdk: AcuantJavascriptWebSdkInterface;
+    /* Possible AcuantCamera on the window object (11.5.0) */
+    AcuantCamera: AcuantCameraInterface;
+  }
+}
+
+interface AcuantContextProviderProps {
+  sdkSrc: string; // SDK source URL.
+  cameraSrc: string; // Camera JavaScript source URL.
+  credentials: string | null; // SDK credentials.
+  endpoint: string; // Endpoint to submit payload.
+  glareThreshold: number; // Minimum acceptable glare score for images.
+  sharpnessThreshold: number; // Minimum acceptable sharpness score for images.
+  children: ReactNode; // Child element
+}
+
+/**
+ * The minimum glare score value to be considered acceptable.
+ *
+ */
+export const DEFAULT_ACCEPTABLE_GLARE_SCORE = 30;
+
+/**
+ * The minimum sharpness score value to be considered acceptable.
+ *
+ */
+export const DEFAULT_ACCEPTABLE_SHARPNESS_SCORE = 30;
+
+/**
+ * Returns the containing directory of the given file, including a trailing slash.
+ */
+export const dirname = (file: string): string => {
+  return file.split('/').slice(0, -1).concat('').join('/');
+};
+
+interface AcuantContextInterface {
+  isReady: boolean;
+  isAcuantLoaded: boolean;
+  isError: boolean;
+  isCameraSupported: boolean | null;
+  isActive: boolean;
+  setIsActive: (nextIsActive: boolean) => void;
+  credentials: string | null;
+  glareThreshold: number;
+  sharpnessThreshold: number;
+  endpoint: string | null;
+}
+
+const AcuantContext = createContext<AcuantContextInterface>({
+  isReady: false,
+  isAcuantLoaded: false,
+  isError: false,
+  isCameraSupported: /** @type {boolean?} */ null,
+  isActive: false,
+  setIsActive: () => {},
+  credentials: null,
+  glareThreshold: DEFAULT_ACCEPTABLE_GLARE_SCORE,
+  sharpnessThreshold: DEFAULT_ACCEPTABLE_SHARPNESS_SCORE,
+  endpoint: /** @type {string?} */ null,
+});
+
+AcuantContext.displayName = 'AcuantContext';
+
+/**
+ * Returns a found AcuantJavascriptWebSdk
+ * object, if one is available.
+ * This function normalizes differences between
+ * the 11.5.0 and 11.7.0 SDKs. The former attached
+ * the object to the global window, while the latter
+ * sets the object in the global (but non-window)
+ * scope.
+ */
+const getActualAcuantJavascriptWebSdk = (): AcuantJavascriptWebSdkInterface => {
+  if (window.AcuantJavascriptWebSdk) {
+    return window.AcuantJavascriptWebSdk;
+  }
+  return AcuantJavascriptWebSdk;
```

Comment:
If the scripts failed to load for some reason, trying to access the global this way could throw an error. I'm not sure if that's too much of a problem if we're likely already throwing errors anyways in this circumstance, but it might be good to add a guard to be safe.

![image](https://user-images.githubusercontent.com/1779930/181596918-2b5d58d0-900b-4f60-a4fb-2b0768a2c05b.png)

```suggestion
  if (typeof AcuantJavascriptWebSdk !== 'undefined') {
    return AcuantJavascriptWebSdk;
  }
```

Reply:
Updated in b563c26